### ISSUE TITLE
In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator is not visible on 'Select a product' drop down menu items in windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/ajax-docs#831]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.7271)
Browser: Edge (Version 144.0.3719.18)
URL: https://demos.telerik.com/aspnet-ajax/dropdownlist/examples/overview/defaultcs.aspx

**Pre Requisites:**
Setting > Accessibility > Contrast themes > Aquatic/Desert/Dusk/Night Sky

**Repro Steps:**

Open the URL: [https://demos. in telerik.com/aspnet-ajax/dropdownlist/examples/overview/defaultcs.aspx](https://demos.telerik.com/aspnet-ajax/dropdownlist/examples/overview/defaultcs.aspx) in edge browser in windows.
Press tab key to reach to the 'Select a product' drop down and expand it.
Drop down list item and selected list item will appear.
Verify whether In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator is visible on drop down menu items and selected list item in windows or not.
Actual Result:
In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator is not visible on 'Select a product' drop down menu items in windows.

**Refer Attachment:**

<img width="1937" height="1037" alt="Image" src="https://github.com/user-attachments/assets/f2b2cff0-6404-4f1c-a729-fa3c11e5ed82" />

https://github.com/user-attachments/assets/f2506b5f-86ea-48fe-9ef8-2ac137c3f852

**Expected Result:**
In High contrast Aquatic/Dusk/Desert and Night sky themes, keyboard focus indicator should be visible properly on 'Select a product' drop down menu items in windows.
